runelite-client/src/main/java/net/runelite/client/plugins/keyremapping/KeyRemappingListener.java

```diff
@@ -308,6 +313,10 @@ else if (config.esc().matches(e))
 				{
 					mappedKeyCode = KeyEvent.VK_ESCAPE;
 				}
+				else if (config.space().matches(e))
```

Comment:
It might be good to write a test for this, since it would show you this is incorrect as it only remaps when fkeys is on.

Reply:
👀